cachito/web/api_v1.py

```diff
@@ -28,15 +30,23 @@ def get_requests():
         value exceeds configuration's CACHITO_MAX_PER_PAGE.
     :rtype: flask.Response
     """
+    # Check if user is filtering requests by state
+    state = flask.request.args.get('state', default='')
```

Comment:
The more "Pythonic" approach would be to let `state` be `None` if it wasn't in the request.